aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -81,11 +81,29 @@ void div_kernel(TensorIterator& iter) {
   }
 }
 
+template <class self_t, class other_t>
+void logical_xor_kernel_impl(TensorIterator& iter) {
+  AT_DISPATCH_ALL_TYPES_AND(kBool, iter.dtype(0), "logical_xor_cpu", [&]() {
+    cpu_kernel(iter,
+      [](self_t a, other_t b) -> scalar_t {
+        return static_cast<scalar_t>((!a) != (!b));
```

Comment:
this seems fine, just curious why you didn't convert to bool rather than using "!"?

Reply:
I guess I got some foreign language influence :) Fixed now.